source/docs/application-notes/devblog.rst

```diff
@@ -0,0 +1,13 @@
+Devblog Resources
+=================
+
+This is a list of devblog resources that are meant to be used for particular applications.
```

Comment:
This is kinda confusing. Maybe change to
"Welcome to the development blog. Here, we will highlight various features of CTR Electronics devices and how they can be utilized in specific applications. "